I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script1.rpy", line 747, in script
    $ sceneskip()
  File "game/script1.rpy", line 747, in <module>
    $ sceneskip()
TypeError: 'bool' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script1.rpy", line 747, in script
    $ sceneskip()
  File "/Users/jayokuo/Desktop/renpy-8.3.3-sdk/renpy/ast.py", line 821, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Users/jayokuo/Desktop/renpy-8.3.3-sdk/renpy/python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/script1.rpy", line 747, in <module>
    $ sceneskip()
TypeError: 'bool' object is not callable

macOS-14.5-arm64-arm-64bit arm64
Ren'Py 8.3.3.24111502
3 More Days 1.2
Sun Jun  1 23:12:27 2025
